clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -79,14 +79,18 @@ object ElasticsearchIndex extends Elastic4sAutoDerivation {
     )
 
   implicit val Gvcf: ElasticsearchIndex[DocumentGvcf] =
-    // despite being decoupled from "v1", we append -v2 to keep ES indices consistent with GCS
+    /* Despite being decoupled from "v1", we append -v2 to keep ES indices consistent with GCS.
```

Comment:
I think our usual convention is to have the first line of text in the second line of a `/*` `*/` block comment. You could use a `//` comment here instead since it doesn't require a blank line before and after.